optimade/models/jsonapi.py

```diff
@@ -137,8 +137,11 @@ class BaseResource(BaseModel):
 
 
 class RelationshipLinks(BaseModel):
-    """A resource object **MAY** contain references to other resource objects (\"relationships\").
-    Relationships may be to-one or to-many. Relationships can be specified by including a member in a resource's links object."""
+    """ A resource object **MAY** contain references to other resource objects (\"relationships\").
+    Relationships may be to-one or to-many.
+    Relationships can be specified by including a member in a resource's links object.
```

Comment:
I had a hard time seeing where this comes from, but I can trace it back to the JSON API spec. Except the last sentence.

Reply:
Yep, a lot of this part was documented by whoever wrote these models, didn't want to change it so much in this PR

Reply:
Probably us 😅 but yeah. Just wanted to make sure when we're using `MAY` and such.

Reply:
I've replaced this entire docstring with the link you suggested in your other comment.